Makefile

```diff
@@ -1,34 +1,52 @@
-DOCKER_REPO ?= gcr.io/online-bridge-hackathon-2020
+# Project / Org
+GCP_PROJECT ?= globalbridge-app
+GKE_ZONE ?= europe-west4-b
+
+# Service / App
+GKE_CLUSTER_NAME ?= prod-cluster
+RELEASE_NAME ?= deal
+K8S_NS ?= prod-${RELEASE_NAME}
+EXTERNAL_ADDRESS ?= ${RELEASE_NAME}.prod.globalbridge.app
+
+# Docker Config
+DOCKER_REPO ?= gcr.io/${GCP_PROJECT}
 VERSION ?= $(shell cat VERSION)
-DOCKER_TAG=${DOCKER_REPO}/deal-api:${VERSION}
+DOCKER_TAG = ${DOCKER_REPO}/${RELEASE_NAME}:${VERSION}
 
-EXTERNAL_ADDRES ?= deal.hackathon.globalbridge.app
 
-DDS_K8S_NS ?= deal-api
-GCP_PROJECT ?= online-bridge-hackathon-2020
-GKE_CLUSTER_NAME ?= hackathon-cluster
-GKE_ZONE ?= europe-west3-b
+ifeq (${SILENT},1)
+	VERBOSE_TEST :=
+else
+	VERBOSE_TEST := -v
+endif
+
 
 release: build push
 
-build:
+libdds/.git:
+	git submodule update --init
+
```

Comment:
Copy paste code which doesn't work correctly. Something similar would be useful but it requires a bit different syntax. I think it would be better to it for now.

Reply:
removed